.github/workflows/test.yml

```diff
@@ -97,11 +97,20 @@ jobs:
         if: matrix.runner != 'windows-latest'
         run: echo | openssl s_client -showcerts -servername 'api.vercel.com' -connect 76.76.21.21:443
 
-  conclusion:
+  summary:
+    name: Summary
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    if: always()
     needs:
       - test
-    runs-on: ubuntu-latest
-    name: E2E
     steps:
-      - name: Done
-        run: echo "Done."
+      - run: |-
```

Comment:
Should we add a name? Currently it shows as "Run for status in failure" or "Run for status in success"

Reply:
Fixed in [2ff02a8](https://github.com/vercel/vercel/pull/10048/commits/2ff02a83a9200842e9fd81025469d5c054a56d4f)